acceptance/acceptance_test.go

```diff
@@ -64,7 +64,10 @@ func lifecycleCmd(t *testing.T, name string, args ...string) *exec.Cmd {
 	cmdArgs := append(
 		[]string{
 			"run",
-			"-ldflags", "-X github.com/buildpack/lifecycle/cmd.buildVersion=some-version",
+			"-ldflags",
+			"-X github.com/buildpack/lifecycle.Version=some-version " +
+				"-X github.com/buildpack/lifecycle.SCMCommit=asdf123 " +
+				"-X github.com/buildpack/lifecycle.SCMRepository=github.com/buildpack/lifecycle",
```

Comment:
minor nitpick: if we aren't asserting on anything that uses SCMRepository it probably doesn't make sense to set it for the tests